src/components/ZclAttributeManager.vue

```diff
@@ -185,7 +185,12 @@ limitations under the License.
                   : ''
               "
               :disable="isDisabledDefault(props.row.id, selectedCluster.id)"
-              :model-value="defaultValueCheck(props.row.id, selectedCluster.id)"
+              :model-value="
+                props.row.isNullable &&
+                defaultValueCheck(props.row.id, selectedCluster.id) === null
+                  ? 'NULL'
```

Comment:
I am not happy that we hardcode "NULL" here. This is mixing up the C concept of "NULL" with a concept of "nil" from the spec.

What if we're not generating for C, so we don't have "NULL"?